monkey/monkey_island/cc/environment/test_aws.py

```diff
@@ -7,7 +8,7 @@
 
 class TestAwsEnvironment(IslandTestCase):
```

Comment:
We're changing the AWS env behaviour to be default, so this test might become redundant - if so delete it : )

Reply:
good call